modules/container_deployment/prom_values.yaml

```diff
@@ -0,0 +1,61 @@
+grafana:
+  service:
+    type: "LoadBalancer"
+prometheusOperator:
+  service:
+    type: "LoadBalancer"
+prometheus:
+  service:
+    type: "LoadBalancer"
+  prometheusSpec:
+    additionalScrapeConfigs:
+      - job_name: hono-pods
+        honor_timestamps: true
+        scrape_interval: 10s
+        scrape_timeout: 10s
+        metrics_path: /metrics
+        scheme: https
+        kubernetes_sd_configs:
+        - role: pod
+          namespaces:
+            names:
+            - default
+          selectors:
+          - role: pod
+            label: app.kubernetes.io/instance=hono,helm.sh/chart=hono-1.5.10
+        tls_config:
+          insecure_skip_verify: true
+        relabel_configs:
+        - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
+          separator: ;
+          regex: "true"
+          replacement: $1
+          action: keep
+        - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_path]
+          separator: ;
+          regex: (.+)
+          target_label: __metrics_path__
+          replacement: $1
+          action: replace
+        - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
+          separator: ;
+          regex: ([^:]+)(?::\d+)?;(\d+)
+          target_label: __address__
+          replacement: $1:$2
+          action: replace
+        - separator: ;
+          regex: __meta_kubernetes_pod_label_(.+)
+          replacement: $1
+          action: labelmap
+        - source_labels: [__meta_kubernetes_namespace]
+          separator: ;
+          regex: (.*)
+          target_label: kubernetes_namespace
+          replacement: $1
+          action: replace
+        - source_labels: [__meta_kubernetes_pod_name]
+          separator: ;
+          regex: (.*)
+          target_label: kubernetes_pod_name
+          replacement: $1
+          action: replace
```

Comment:
Should some links to documentation regarding to relabeling and kubernetes __meta_ values be included with documentation:
https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config
https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kubernetes_sd_config

On the other hand, these are Prometheus specific and not needed if one does not intent to add more stuff to these deployment scripts. Might be useful to describe a how one adds something to Prometheus monitoring. If there is documentation available in the https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack point to this. 
In minimum make reference to https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack for more information.

Reply:
Just to clarify documentation would probably not be included in this specific file, but as general documentation about Prometheus usage in smad-deploy-azure.